sdk/src/main/java/io/dapr/exceptions/DaprError.java

```diff
@@ -5,9 +5,13 @@
 
 package io.dapr.exceptions;
 
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import io.grpc.Status;
+
 /**
  * Represents an error message from Dapr.
  */
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
```

Comment:
Will it be better to have getter/setter for code instead of this annotation? To make it consistent?

Reply:
The internal property is private. I don't want to expose that on purpose.